### PR TITLE
Testing Popularity Recommendation System

### PR DESCRIPTION
**Pull request by 5ukhy21**
Created 3 tests for all three functions that make up the Popularity Recommendation System (str, calculate_artist_avg_popularity, and recommend_songs).
Additionally, updating the README file to outline and provide instructions on the test files provided.

### CODE REVIEW

**Comment by Yer1k** [comment]
Can you please post the screenshots of passing pytest and coverage thing? @5ukhy21

File: tests/test_popularity_recommender.py [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+            "James Levine": 5,
+            "Berliner Philharmoniker": 5,
+            "Dingkun Yang": 3,
+            "Pragya R": 3,
+            "Sukhpreet S": 6,
+        }
+        assert s.recommend_songs(2) == [
+            "Dingkun is a genius",
+            "Piano Concerto No. 3 in C Major",
+        ]
```
Comment (Yer1k) [inline tests/test_popularity_recommender.py:72]: Thanks for saying that. : )

**Comment by 5ukhy21** [comment]
Hey @Yer1k
See the screenshots below.
The popularity recommender passed and with 100% coverage! While the parse_data is lower in coverage, this is being testing using the test_parse_data file

image

image

File: tests/test_popularity_recommender.py [hunk 2]
```diff
@@ -0,0 +1,77 @@
+"""Test Song Recommendation functions."""
+import pytest
+
```
Comment (Yer1k) [inline tests/test_popularity_recommender.py]: As for the format, we want to have two blank lines after that, and for this, we do not necessarily need pytest to get imported. Have updated in this thread

**Review by Yer1k — APPROVED** [review]
Looks good to me

File: tests/test_popularity_recommender.py [hunk 3]
```diff
@@ -1,5 +1,5 @@
 """Test Song Recommendation functions."""
-import pytest
+
```
Comment (pr-124) [inline tests/test_popularity_recommender.py]: I agree with this.

**Review by pr-124 — APPROVED** [review]
Looks good to merge

**Pull request merged**